go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.2
+	github.com/tetratelabs/wazero v1.0.0-rc.1
```

Comment:
Can we do a before and after comparison of the binary size?

Reply:
Should we bump to rc.2 now?

Reply:
I don't love that this adds 3MB but on balance, probably nothing we can do. We should evaluate if there's places we can save on our binary size as it's starting to get a bit big